gtest/src/manager.rs

```diff
@@ -445,6 +449,61 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn execute_custom_function(
```

Comment:
I'd rename the functions. 
```suggestion
    pub(crate) fn execute_with_result(
```